### PR TITLE
Configured for consortia

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose
To configure consortia for busybee.
## Approach

- Added custom env variables for mod-consortia and authtoken modules work properly.
- Added custom config file for only acq-modules and mod-consorta setup. (it is removed unused modules)

### CODE REVIEW

File: config-thunderjet.yml
Comment (SerhiiNosko) [inline config-thunderjet.yml]: if we add folio_consortia_settings - we will be able to run consortia UI also.
But we can do it in subsequent PR

File: busybee/service.py
Comment (azizbekxm) [inline busybee/service.py:168]: contains main logic, Other places reformatted based on official python style. refactoring part can be reverted, if you want.

File: config-thunderjet.yml
Comment (SerhiiNosko) [inline config-thunderjet.yml:22]: mod-source-record-storage and mod-source-record-manager need for Data Import functionality, so we can remove from Thunderjet config to speedup deployment time.
mod-pubsub also not needed

File: config-thunderjet.yml
Comment (damien-git) [reply config-thunderjet.yml]: For the UI the following are also useful:
- in be-modules: mod-agreements
- in ui-modules: folio_acquisition-units, folio_developer

File: config-thunderjet.yml
Comment (SerhiiNosko) [reply config-thunderjet.yml:22]: edge-orders, mod-ebsconet, mod-gobi better to have to be able to run all our karate tests

Comment (SerhiiNosko) [reply config-thunderjet.yml:22]: we also own functionality for Edifact exports and have karate tests for it, so can include these modules in config:
mod-data-export-spring, mod-data-export-worker but also we need to figure out how to setup MinIO, so to not block this PR lets do this in separate PR

Comment (azizbekxm) [reply config-thunderjet.yml:22]: mod-source-record-storage can be removed, need to check, but source-record-manager currently is being used by consortia. pubsub also being used by orders. Also thanks I will add edge-order, ebsconet gobi

File: busybee/service.py
Comment (SerhiiNosko) [reply busybee/service.py:168]: we have more that 7 modules that use system users, so in order to not modify source code each time we can create a file where specify env varaibles for each module and read that file here. But again to not block current consortia karate stories we can do it later, as @okolawole-ebsco prefer

File: config-thunderjet.yml
Comment (SerhiiNosko) [inline config-thunderjet.yml]: please update comment 
add this module to install.json file or specify module in additional-json-path that will be possible after this PR merged:
https://github.com/Olamshin/busybee/pull/4/files

File: config-thunderjet.yml
Comment (azizbekxm) [reply config-thunderjet.yml]: most guys will only need consorita (rare) and order/finance/invoice. To be fast setup So I will include as comment.

File: config-thunderjet.yml
Comment (azizbekxm) [reply config-thunderjet.yml]: Updated

**Review by SerhiiNosko — APPROVED** [review]

File: busybee/service.py
Comment (Olamshin) [reply busybee/service.py:168]: module specifc env vars is something I had thought about. I like the approach you describe @SerhiiNosko

Comment (Olamshin) [reply busybee/service.py:168]: @azizbekxm some of the refactoring don't make sense i think that may be because of macOS/Linux line endings. We can reformat all the code according to a specific linter config defined in this repo later.

File: busybee/service.py
Comment (Olamshin) [inline busybee/service.py:162]: nitpick: since module_id is a string and not a list, using `in` instead of `==` seems off.

Comment (azizbekxm) [reply busybee/service.py:162]: As far as I know, in python we can use string.contains method of java as `in` operation. Regarding why I am not using `==`, module name contains snapshot version, so need to use contains operation.

reference first section
https://realpython.com/python-string-contains-substring/

File: busybee/service.py
Comment (azizbekxm) [reply busybee/service.py:168]: I got it, reverted.

File: busybee/service.py
Comment (Olamshin) [reply busybee/service.py:162]: I see so it was more of a contains operator. Got it!

**Review by Olamshin — APPROVED** [review]

**Pull request merged**
